.code-samples.meilisearch.yaml

```diff
@@ -780,12 +780,10 @@ multi_search_federated_1: |-
       {
         indexUid: 'movies',
         q: 'batman',
-        limit: 5,
```

Comment:
Optionally, limit can be moved where it belongs, inside `federation`, instead of being removed. But this is also fine.

Reply:
We have to follow the same example than in the docs, I linked in the PR 😊